src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="/styles/index.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="Photo of Apple iMac 27"
+      >
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">
+        <span>Product code:</span>
+        <span data-id="id">195434</span>
+      </div>
+
+      <div class="card__rating">
+
+        <div class="stars stars--4">
+          <div class="stars__star star1"></div>
```

Comment:
```suggestion
          <div class="stars__star"></div>
```

Reply:
1 - this star is an element of stars block, not a separate BEM block
2 -  why do you need this numbers at the and of class, use pseudoelements to style elements according to their position in parent element

Reply:
Oh, just saw that i've mistaken, I meant pseudoclasses, sorry😅

Reply:
It`s alright)